<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CARI DATA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#4A646C;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 50pt &quot;Juice ITC&quot;;</string>
    </property>
    <property name="text">
     <string>DATA PELANGGAN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_cari">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white; color:black;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Masukkan nota yang ingin dicari</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cari">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CARI</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updatestatus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>UPDATE </string>
    </property>
   </widget>
   <widget class="QPushButton" name="tambah">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TAMBAH DATA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_total">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>460</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TOTAL</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
